scripts/gulp/tasks/build/compile/ts-configs.ts

```diff
@@ -19,7 +19,8 @@ export function createTsConfigEsm2015(packageName): string {
     "skipMetadataEmit": false,
     "strictTypeChecks": true,
     "flatModuleOutFile": "index.js",
-    "flatModuleId": "@nebular/${packageName}"
+    "flatModuleId": "@nebular/${packageName}",
+    "enableIvy": false
```

Comment:
we need to compile libs to viewEngine and them user's project will run ngcc right?

Reply:
Yep